meterfs/meterfs.c

```diff
@@ -633,6 +728,9 @@ int meterfs_writeFile(id_t id, const char *buff, size_t bufflen, meterfs_ctx_t *
 	file_t *f;
 	int err;
 
+	if (bufflen <= 0)
```

Comment:
```suggestion
	if (bufflen == 0)
```
`bufflen` is type `size_t` and so `>= 0`.